### ISSUE TITLE
Loading spinner flashes scrollbar

### ISSUE THREAD

**Issue by superhawk610** [picturepan2/spectre#394]
Loading spinner containers display a strange behavior as the mouse moves across the screen, flashing the scrollbar on and off:

![stutter](https://user-images.githubusercontent.com/18172185/35643090-2ff5ee9a-0693-11e8-867c-6b8943c85373.gif)

This seems to be caused by them barely overflowing the container and then shrinking back inside. Adding `overflow: hidden` to `.loading` fixes this behavior:

![fixed](https://user-images.githubusercontent.com/18172185/35643117-4da0dee6-0693-11e8-8405-a341ccae6ab4.gif)

I've opened a [PR](https://github.com/picturepan2/spectre/pull/395) that adds this fix.

**Comment by picturepan2** [comment]
@superhawk610 Can you show me your code?

**Comment by superhawk610** [comment]
I've created a [Codepen ](https://codepen.io/superhawk610/pen/PQEVOO) to illustrate the issue, but in doing so I realized that this issue is not due to Spectre.css but instead must have been some conflict with another library I am using, because the component on Codepen works fine and does not display this issue. I've closed my PR. If I can isolate the cause of the original issue, I will update this Issue, but the project has progressed quite a bit since then and I am currently unable to reproduce the issue.
